thunder/executors/nvfuserex_impl.py

```diff
@@ -144,7 +148,16 @@ def is_supported_devicetype(devicetype: DeviceType) -> bool:
     return devicetype is DeviceType.CUDA
 
 
-_low_precision_floats = (dtypes.float16, dtypes.float16_, dtypes.bfloat16, dtypes.bfloat16_)
+_low_precision_floats = (
+    dtypes.float16,
+    dtypes.float16_,
+    dtypes.bfloat16,
+    dtypes.bfloat16_,
+    dtypes.float8_e5m2,
+    dtypes.float8_e5m2_,
+    dtypes.float8_e4m3fn,
+    dtypes.float8_e4m3fn_,
+)
 
 
 def is_supported_dtype(dtype: type | dtypes.dtype, *, allow_low_precision_floats: bool = True) -> bool:
```

Comment:
I think it necessary to update this function to check if a device is >= 90 when dtype is fp8 as per  https://github.com/NVIDIA/Fuser/blob/79a088abe09fb746c2df792e03c0cce738dd36e7/csrc/device_lower/analysis/device_version.cpp#L20-L25